homeassistant/components/neato/__init__.py

```diff
@@ -161,7 +160,7 @@ def login(self):
         self.logged_in = True
         _LOGGER.debug("Successfully connected to Neato API")
 
-    @Throttle(timedelta(minutes=SCAN_INTERVAL_MINUTES))
+    @Throttle(timedelta(minutes=1))
```

Comment:
So I was wondering if we should keep this as `minutes` or convert to `seconds` so users can override and set scan interval based on seconds instead of minutes.  What do you think?  

Also should we consider changing the default scan interval to match what we are setting here?

Reply:
I don't think that this could be overwritten at all since this isn't bound to any platform. Correct me, if I'm wrong. That's why I want to lower this as much as possible. The other platforms try to poll the API with their individual scan interval. And we want to cap there when the maximum is reached.

Reply:
Ah your right I was thinking about the scan interval we set on the platforms.  This is a good catch then.